sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16655,8 +16655,9 @@
               "id": "e9yrlbn",
               "inputType": "text",
               "inputMask": "",
-              "decimalLimit": 2,
-              "requireDecimal": true
+              "decimalLimit": 0,
```

Comment:
I am not able to follow how did we get these properties. 

I checked the currency component in our form builder and the enhanced form builder in https://formio.github.io/formio.js/app/sandbox.html and I couldnt see any properties related to this in the component.

Please let me know if I am missing something here.